src/Compilers/Core/Portable/ReferenceManager/BoundInputAssembly.cs

```diff
@@ -31,9 +33,9 @@ internal struct BoundInputAssembly
             /// Each element describes the assembly the corresponding reference of the input assembly 
             /// is bound to.
             /// </remarks>
-            internal AssemblyReferenceBinding[] ReferenceBinding;
+            internal AssemblyReferenceBinding[]? ReferenceBinding;
 
-            private string GetDebuggerDisplay()
+            private string? GetDebuggerDisplay()
```

Comment:
Just curious, how does the debugger behave when the method given to `DebuggerDisplay` returns `null`?

Reply:
For the way it's defined on this type, using the `[DebuggerDisplay]` attribute, it will display simply `null`.